example/main.go

```diff
@@ -0,0 +1,64 @@
+package main

```

Comment:
@ZachPlunkett  Thank you so much for example/main.go. This was needed. Great work with the ideation.

Reply:
Thank you for the feedback!